types/BillingInfo.js

```diff
@@ -35,4 +35,17 @@ const DonationInfoPublic = S.object()
     S.number().description('Next charge date in milliseconds timestamp')
   )
 
-module.exports = { BillingInfo, BillingInfoUpdate, BillingInfoWithCustomerId, DonationInfoPublic }
+const OrgDonationInfoPublic = S.object()
+  .extend(DonationInfoPublic)
+  .prop(
+    'totalDonationsMade',
```

Comment:
nit: maybe `totalDonated` or `totalAmountDonated`? it's a very smol nit. just `totalDonationsMade` sounds wrong to me.